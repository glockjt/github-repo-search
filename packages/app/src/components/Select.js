import styled from "@emotion/styled";

const Select = styled.select`
  padding: 8px;
  background-color: #fff;
  font-size: 16px;
  border: 0;
`;

export default Select;
