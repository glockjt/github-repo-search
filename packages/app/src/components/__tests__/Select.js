import React from "react";
import renderer from "react-test-renderer";
import Select from "../Select";

describe("Select Component Tests", () => {
  it("renders correctly with no options", () => {
    const tree = renderer.create(<Select />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it("renders correctly with options", () => {
    const tree = renderer
      .create(
        <Select>
          <option>Test Item</option>
        </Select>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
