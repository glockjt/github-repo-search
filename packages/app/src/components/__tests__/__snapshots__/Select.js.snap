// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select Component Tests renders correctly with no options 1`] = `
<select
  className="css-1x5yj64"
/>
`;

exports[`Select Component Tests renders correctly with options 1`] = `
<select
  className="css-1x5yj64"
>
  <option>
    Test Item
  </option>
</select>
`;
